lib/taiko.js

```diff
@@ -2042,13 +2043,18 @@ module.exports.image = (attrValuePairs, ...args) => {
  * @param {...relativeSelector} args - Proximity selectors
  * @returns {ElementWrapper}
  */
-module.exports.link = (attrValuePairs, ...args) => {
+module.exports.link = (attrValuePairs, _options = {}, ...args) => {
   validate();
-  const { selector } = prepareParameters(attrValuePairs, ...args);
+  const { selector, options } = prepareParameters(
+    attrValuePairs,
+    _options,
+    ...args,
+  );
   const get = getElementGetter(
     selector,
-    async () => await match(selector.label).get('a', 0, 0),
+    async () => await match(selector.label, options).get('a', 0, 0),
```

Comment:
That is true too. In many of the other API's we have a lot of fields that is not needed by the match function. Also, since `match` is only used internally, we can always change the API to include more options or pass the whole object later. I don't really see a need for this in the near future though.

Reply:
As requested, changed the code to pass SelectHiddenElement field directly.